firefoxThemes/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Doki Theme for Firefox",
   "short_name": "Doki Theme",
-  "version": "18.0.0",
+  "version": "18.1.0",
```

Comment:
Although updated here, the `package.json` file also needs a version update too.

Reply:
Yeah, I didn't do that because that updates all the things in all the places, I might need to re-think how versioning is handled for a repo that contains distinct products. Might be that FireFox gets its own repository, I dunno yet.

Reply:
I agree. The Firefox component should have its own separate repo considering how dynamic the theming system is compared to the chromiums. Also considering the fact that majority of recent pull requests are focused around Firefox as well.

Reply:
Yeah I'll do that later once I prepare another theme release.